addons/point_of_sale/static/src/app/screens/product_screen/product_list/product_list.js

```diff
@@ -44,6 +44,19 @@ export class ProductsWidget extends Component {
             observer.observe(productsWidget);
             return () => observer.disconnect();
         });
+        const toggleIsMobile = () => this.toggleIsMobile();
+        onMounted(() => {
+            toggleIsMobile();
+            window.addEventListener("resize", toggleIsMobile);
+        });
+        onWillUnmount(() => {
+            window.removeEventListener("resize", toggleIsMobile);
+        });
```

Comment:
```suggestion
        useExternalListener(window, "resize", toggleIsMobile);
        onMounted(toggleIsMobile);
```

May I know why `ui.isSmall` is enough as the condition to add the 'scrollbar-hidden' class?